docs/data/date-pickers/date-time-range-field/BasicDateTimeRangeField.tsx

```diff
@@ -1,15 +1,12 @@
 import * as React from 'react';
-import Stack from '@mui/material/Stack';
 import { LocalizationProvider } from '@mui/x-date-pickers';
 import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
 import { Unstable_MultiInputDateTimeRangeField as MultiInputDateTimeRangeField } from '@mui/x-date-pickers-pro/MultiInputDateTimeRangeField';
 
 export default function BasicDateTimeRangeField() {
   return (
     <LocalizationProvider dateAdapter={AdapterDayjs}>
-      <Stack spacing={4}>
-        <MultiInputDateTimeRangeField />
-      </Stack>
+      <MultiInputDateTimeRangeField />
```

Comment:
Ideally this demo could use a bit more width as well.
![Screenshot 2022-12-09 at 15 19 04](https://user-images.githubusercontent.com/4941090/206711277-0a161dd0-2f37-46f3-92b3-867d365ce2fd.png)

Reply:
See https://github.com/mui/mui-x/pull/7149#discussion_r1045490074